src/app/containers/ArticleFigure/index.jsx

```diff
@@ -14,10 +14,10 @@ import {
 } from '../../lib/styledGrid';
 
 const renderCopyright = copyright =>
-  copyright ? <Copyright>{copyright}</Copyright> : null;
```

Comment:
Are this and the following changes required to resolve the issue?

Reply:
Not really, Just noticed i could make those lines much better while investigating the issue. The changes here do not directly affect the issue